app/DTO/Rights/InitRightsDTO.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\DTO\Rights;
+
+use App\DTO\ArrayableDTO;
+
+/**
+ * This DTO provides the information whether some actions are available to the user.
+ */
+class InitRightsDTO extends ArrayableDTO
+{
+	public function __construct(
+		public RootAlbumRightsDTO $root_album,
+		public SettingsRightsDTO $settings,
+		public UserRightsDTO $users
+	) {
+	}
+
+	/**
+	 * Create from current user.
+	 *
+	 * @return self
+	 */
+	public static function ofCurrentUser(): self
+	{
+		return new self(
+			root_album: RootAlbumRightsDTO::ofCurrentUser(),
+			settings: SettingsRightsDTO::ofCurrentUser(),
+			users: UserRightsDTO::ofCurrentUser()
+		);
+	}
+
+	/**
+	 * Create from no admin registerd.
+	 *
+	 * @return self
+	 */
+	public static function ofAdminIsNotRegistered(): self
```

Comment:
```suggestion
	public static function ofUnregisteredAdmin(): self
```

In order to stay consistent with `ofCurrentUser` and again `ofAdminIsNotRegistered` is not grammatically correct.

BTW: We sometimes use the term "registered" and sometimes "configured", i.e. on the one hand side we have the "(not |un-)registered admin" and the "(not |un-)configured admin" on the other hand side. I have no preference for either option, but we should be consistent, decide wether the admin account is registered or configured and then stick to it.

Reply:
Fixed [31a5e2b](https://github.com/LycheeOrg/Lychee/pull/1539/commits/31a5e2b97c5302595e7b13326c54d6980c16498b)